x-pack/plugins/monitoring/server/alerts/cpu_usage_alert.test.ts

```diff
@@ -192,13 +190,14 @@ describe('CpuUsageAlert', () => {
         ],
       });
       expect(scheduleActions).toHaveBeenCalledWith('default', {
-        internalFullMessage: `CPU usage alert is firing for ${count} node(s) in cluster: ${clusterName}. [View nodes](elasticsearch/nodes)`,
-        internalShortMessage: `CPU usage alert is firing for ${count} node(s) in cluster: ${clusterName}. Verify CPU levels across affected nodes.`,
-        action: `[View nodes](elasticsearch/nodes)`,
-        actionPlain: 'Verify CPU levels across affected nodes.',
+        internalFullMessage: `CPU usage alert is firing for node ${nodeName} in cluster: ${clusterName}. [View node](http://localhost:5601/app/monitoring#/elasticsearch/nodes/${nodeId}?_g=(cluster_uuid:${clusterUuid}))`,
```

Comment:
Is the idea that `[View node]` will link you back to the node page? Should we be using `http://localhost:5601` or is there a way to inject the current URL?